{"name": "test_start", "status": "broken", "statusDetails": {"message": "AttributeError: module 'Webapi.keys' has no attribute 'post'", "trace": "self = <ddt.yuanting_test.Test_Yuanting object at 0x1025f5850>\n\n    @allure.story('申请OA')\n    # @pytest.mark.parametrize('listcases', datas['workflow'])\n    # def test_comm(self,listcases):\n    #     allure.dynamic.title(listcases['title'])\n    #     allure.dynamic.description(listcases['description'])\n    #     testcases = listcases['cases']\n    #     print(testcases)\n    #     for cases in testcases:\n    #         listcase = list(cases.values())\n    #         print(listcase)\n    #         url=listcase[3]\n    #         datas=listcase[2]\n    #         with allure.step(listcase[0]):\n    #             ree=keys.post_test(url,datas)\n    \n    \n    \n    # def test_login(self):\n    #     self.data=requests.post('https://cdc.test.yuanting.cn/api/passport/login',data={'account':\"ycc000\",'password':\"1111\"})\n    #     self.datas=self.data.json()\n    #     self.datas=self.datas['data']['token']\n    #     return self.datas\n    # def test_start(self):\n    #     self.header=keys.login()\n    #     self.data=requests.post('https://cdc.test.yuanting.cn/api/oa/process/start',data={\"name\":\"[202301172144] ycc000[ycc000]的出差申请\",\"processDefinitionKey\":\"trip_2000\"},headers={\"authorization\":self.header})\n    #     self.datas=self.data.text\n    #     print(self.datas)\n    \n    # def test_login(self):\n    #     self.ree=keys.post('https://cdc.test.yuanting.cn/api/passport/login',data={'account':\"ycc000\",'password':\"1111\"})\n    #     self.datas=self.data.json()\n    #     self.datas=self.datas['data']['token']\n    #     return self.datas\n    def test_start(self):\n        url='https://cdc.test.yuanting.cn/api/oa/process/start'\n        data={\"name\":\"[202301172144] ycc000[ycc000]的出差申请\",\"processDefinitionKey\":\"trip_2000\"}\n>       self.ree=keys.post(url,data)\nE       AttributeError: module 'Webapi.keys' has no attribute 'post'\n\nddt/yuanting_test.py:49: AttributeError"}, "start": 1674388550586, "stop": 1674388550586, "uuid": "b430f4ed-d280-4c1b-80bc-8351a417767b", "historyId": "70c1a5e21cfeac7683a127b632717906", "testCaseId": "03b71f379b6bad0fc66051238e00aee4", "fullName": "ddt.yuanting_test.Test_Yuanting#test_start", "labels": [{"name": "story", "value": "申请OA"}, {"name": "parentSuite", "value": "ddt"}, {"name": "suite", "value": "yuanting_test"}, {"name": "subSuite", "value": "Test_Yuanting"}, {"name": "host", "value": "00demacbook-pro"}, {"name": "thread", "value": "52009-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "ddt.yuanting_test"}]}